python/private/py_wheel.bzl

```diff
@@ -17,6 +17,62 @@
 load("//python/private:stamp.bzl", "is_stamping_enabled")
 load(":py_package.bzl", "py_package_lib")
 
+RequirementsInfo = provider(
+    "Information about all the requirements of a py_wheel.",
+    fields = {
+        "requirements": "The list of requirements.",
+    },
+)
+
+def _requirements_collector_impl(_, ctx):
+    """Aspect to collect third-party dependencies.
+
+    Args:
+        _: The bazel target the aspect is applied to.
+        ctx: The aspect context.
+    Returns: RequirementsInfo
+    """
+
+    requirement = None
+    if hasattr(ctx.rule.attr, "tags"):
```

Comment:
Is this guard here for a reason? tags is part of all rules, so this should always be true. If not, please add a comment about the edge case.

Reply:
Addressed in 49710a0.